src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -367,80 +242,88 @@ move and end listeners for touch and pointer events to the element straight away
-start and move events and request an animation frame in the move event
+start and move events and request an animation frame inside the move event
 callback.
 
-In our demo, we store the initial touch position in `handleGestureStart`:
+In our demo, we store the initial touch position in `handleGestureStart()` (look for `initialTouchPos`):
 
 <pre class="prettyprint">
 {% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/touch-demo-1.html" region_tag="handle-start-gesture" adjust_indentation="auto" %}
 </pre>
 
-The `handleGestureMove` method stores the `y` position before requesting an
-animation frame if we need to, passing in our `onAnimFrame` function as the
+The `handleGestureMove()` method stores the position before requesting an
+animation frame if we need to, passing in our `onAnimFrame()` function as the
 callback:
 
 <pre class="prettyprint">
-{% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/touch-demo-2.html" region_tag="handle-move" adjust_indentation="auto" %}
+{% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/touch-demo-1.html" region_tag="handle-move" adjust_indentation="auto" %}
 </pre>
 
-It’s in the `onAnimFrame` function that we change our UI to move the
-elements around. Initially we check to see if the gesture is still
-on-going to determine whether we should still animate or not, if so we use
-our initial and last y positions to calculate the new transform for our
-element.
+The `onAnimFrame` value is a function that when called, changes our UI
+to move it around. By passing this function into `requestAnimationFrame()`, we
+tell the browser to call it just before it's about to update the page
+(i.e. paint any changes to the page).
+
+In the `handleGestureMove()` callback we initially check if `rafPending` is false,
+which indicates if `onAnimFrame()` has been called by `requestAnimationFrame()`
+since the last move event. Since means we only have one `requestAnimationFrame()`
+waiting to run at any one time.
 
-Once we’ve set the transform, we set the `isAnimating` variable to `false` so
-the next touch event will request a new animation frame.
+When our `onAnimFrame()` callback is executed, we set the transform on any
+elements we want to move before updating `rafPending` to `false`, allowing the
+the next touch event to request a new animation frame.
 
 <pre class="prettyprint">
-{% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/touch-demo-2.html" region_tag="on-anim-frame" adjust_indentation="auto" %}
+{% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/touch-demo-1.html" region_tag="on-anim-frame" adjust_indentation="auto" %}
 </pre>
 
-### Control Gestures using Touch Actions
+### Control gestures using touch actions
 
 The CSS property `touch-action` allows you to control the default touch
 behavior of an element. In our examples, we use `touch-action: none` to
```

Comment:
maybe add a little warning that `touch-action: none` is a bit of a nuclear option, as it will prevent ANY browser default behavior, and that it should be used with caution/surgically on specific elements, as otherwise you end up having to implement scrolling, long-press, etc yourself